templates/project/cargo-generate.toml

```diff
@@ -1,10 +1,10 @@
 [template]
 cargo_generate_version = ">=0.10.0"
 ignore = [".git","README.md"]
-description = "Micro-Rdk template"
+description = "Micro-RDK template"
```

Comment:
Whitespace: at one point I had some other changes to this file but they got reverted, but I kept the ws cleanups